packages/opsian/opsian.0.1/opam

```diff
@@ -9,7 +9,7 @@ bug-reports: "https://github.com/Opsian/opsian-ocaml/issues"
 depends: [
   "dune" {>= "2.8"}
   ("ocaml" {>= "4.12.0" & < "5.0.0"} |
-   ("ocaml" {>= "5.0.0"} & "runtime_events_tools"))
+   ("ocaml" {>= "5.0.0"} & "runtime_events_tools" {< "0.5"}))
```

Comment:
I dont think this'll work. You need a separate line with : `conflicts: [ "runtime_events_tools" {>= "0.5"} ]` and that will ensure this package will never get selected along with runtime_events_tools >= 0.5.  For reasons of solver complexity, you can't have that constraint as part of a conjunction formula I think.

Reply:
Oops, didn't realise that. Updated now.